From be1b4ff0957b287b2d8494967a7f0a1e3401dd8a Mon Sep 17 00:00:00 2001
From: Anna Sztukowska <anna.sztukowska@intel.com>
Date: Mon, 9 Sep 2024 09:36:47 +0200
Subject: [PATCH 170/201] dlink.h: Fix checkpatch warnings for function args

Checkpatch issued a warning due to missing function argument names.
Add the names to resolve the warnings.

Signed-off-by: Anna Sztukowska <anna.sztukowska@intel.com>
---
 dlink.h | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/dlink.h b/dlink.h
index ce667839..18cc294a 100644
--- a/dlink.h
+++ b/dlink.h
@@ -16,11 +16,11 @@ struct __dl_head
 #define dl_prev(p) *(&(((struct __dl_head*)(p))[-1].dh_prev))
 
 void *dl_head(void);
-char *dl_strdup(char *);
-char *dl_strndup(char *, int);
-void dl_insert(void*, void*);
-void dl_add(void*, void*);
-void dl_del(void*);
-void dl_free(void*);
-void dl_init(void*);
+char *dl_strdup(char *s);
+char *dl_strndup(char *s, int l);
+void dl_insert(void *head, void *val);
+void dl_add(void *head, void *val);
+void dl_del(void *val);
+void dl_free(void *v);
+void dl_init(void *v);
 void dl_free_all(void *head);
-- 
2.41.0

